Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+Wallet.swift

```diff
@@ -260,47 +261,52 @@ extension Tab: BraveWalletProviderDelegate {
       self.tabDelegate?.showWalletNotification(self, origin: origin)
     }
   }
+  
+  /// Returns the selected account if present in `accounts`, otherwise returns `accounts`
+  private func filterAccounts(
+    _ accounts: [String],
+    selectedAccount: String?
+  ) -> [String] {
+    if let selectedAccount = selectedAccount, accounts.contains(selectedAccount) {
+      return [selectedAccount]
+    }
+    return accounts
+  }
```

Comment:
can we change this method to `filterAllowedAccounts(_ allowedAccounts: [String], selectedAccount: String?) -> [String]` ? otherwise, i will get confused of what this method is for.

Reply:
Updated in [0702fa0](https://github.com/brave/brave-ios/pull/7483/commits/0702fa059ca6593d4e312ab0d5b2e0e250a031ac).